src/components/dropdown-select.js

```diff
@@ -44,7 +44,9 @@ const DropdownSelect = ({
   data, 
   button, 
   size, 
-  setSelectedOption,
+  uncontrolled,
```

Comment:
_nitpick_: 🤔  maybe instead of explicitly stating uncontrolled as a prop, we could assess based on the presence of `value`?
Not important tho, just trying to reduce number of props lol

Reply:
makes sense, but you did mention: 

> and setSelectedOption prop would be used as a defaultValues prop

i.e., `value` could be present in both cases